imports/plugins/core/versions/server/migrations/4_update_templates_priority.js

```diff
@@ -0,0 +1,40 @@
+import { Migrations } from "/imports/plugins/core/versions";
+import { Packages } from "/lib/collections";
+
+const query = { layout: { $type: 3 } }; // gets docs with layout objects
```

Comment:
OK.. I made a list of the affected packges: `"reaction-accounts", "reaction-checkout", "reaction-dashboard", "reaction-email", "reaction-orders", "reaction-ui", "reaction-inventory", "reaction-product-variant".`

So.. the update will be for only these ones.